CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### 4.2.2 (Next)
 
 * Your contribution here.
+* [#314](https://github.com/alexa-js/alexa-app/pull/314): Fix breaking logic in request.slot(). - [@User1m](https://github.com/user1m)
```

Comment:
More OCD: The period after `user1m).`, check the other lines :)

Reply:
done haha sorry should've been more observant. thanks!